packages/manager/core/generator/app/templates/src/components/ManagerTile/ManagerTile.scss

```diff
@@ -0,0 +1,28 @@
+.manager-tile {
+  .tile {
+    width: 100%;
+
+    .tile-type {
+      display: block;
+      margin-bottom: 16px;
```

Comment:
**intpick:** Use bootstrap utility classes instead

Reply:
we will delete this component with the supercomponent when it will be merged